themes/upbound/assets/scss/_crds.scss

```diff
@@ -0,0 +1,268 @@
+@mixin vertical-bar {
```

Comment:
In order to expand the central display area over the column where the Table of Contents normally is you have to adjust the grid layout. My suggestion is something like:

1. Above this create a new class:
```css
.crd-grid {

  @include media-breakpoint-up(xl) {
  display: grid;
  grid-template-columns: 336px minmax(0,.98fr);
  grid-template-rows: 1fr auto;
  grid-template-areas:
  "left-sidebar-nav content-body right-sidebar-toc"
  "footer-container footer-container footer-container";
  }

}
```

2. Edit `themes/upbound/layouts/_default/baseof.html` and conditionally load the grid:
```diff


    {{ partial "body/top-nav" .}}
-    <div class="grid" data-bs-spy="scroll" data-bs-target="#TableOfContents">
+    <div class="{{ if eq $.Params.Product "crds" }}crd-grid{{else}} grid {{ end }} " data-bs-spy="scroll" data-bs-target="#TableOfContents">
      {{ partialCached "icons/sprites" . }}
      {{ partial "body/left-sidebar-nav" .}}

```